### DISCUSSION TITLE
Why was support for redis-springdata removed in 0.10.1?

### DISCUSSION THREAD

**Discussion by Brutus5000** [MarcGiffing/bucket4j-spring-boot-starter#209] [Q&A] [answered]
The commit message of https://github.com/MarcGiffing/bucket4j-spring-boot-starter/commit/dcebc2fa28e9a9c0e953905ce4f4fe8039be63ff does not state any reason.

**Comment by vladimir-bukhtoyarov** [comment, marked as answer]
I suppose that it is because I removed support for SpringDataRedis https://bucket4j.com/8.2.0/release-notes.html from [Bucket4j](https://github.com/bucket4j/bucket4j)
